### ISSUE TITLE
Trigger new release

### ISSUE THREAD

**Issue by jonsger** [UltraStar-Deluxe/USDX#459]
Hi,

would it be possible to trigger a new release? 

I packaged UltraStar Deluxe for [openSUSE](https://build.opensuse.org/package/show/games/ultrastar-deluxe) and I had to build the package from the current master commit, because the `2017.8` release doesn't recognize the ffmpeg versions on openSUSE. They are to new. With master it works just fine :)

**Comment by deuteragenie** [comment]
I would also love to be able to run fresh Windows and Debian releases.

**Comment by s09bQ5** [comment]
@basisbit, I also feel that it is time to make a release. We always tell people to test if their problem still exists in the current version, but the current release is two years old.

But don't rush things. I'll make a PR this evening that fixes #478.

**Comment by s09bQ5** [comment]
PR #488 done

Btw., https://www.macstadium.com/opensource looks interesting, but requires that "updated builds are released on a regular basis".

**Comment by s09bQ5** [comment]
Which issues deserve to be fixed and which pull request should be merged before the next release?

**Comment by bohning** [comment]
For me, #492 is a show stopper, but I think there is currently not much we can do other than wait for it to be fixed.

**Comment by basisbit** [comment]
#467 also should be done before we create any new build. 
We can add all the issues that should be fixed before a release to the project release board for tracking:
https://github.com/UltraStar-Deluxe/USDX/projects/2

**Comment by nopeppermint** [comment]
any news on planed releases?

**Comment by s09bQ5** [comment]
@basisbit, on https://github.com/UltraStar-Deluxe/USDX/projects/2 you added a todo "fix randomness issues". Can you elaborate on that or provide a list of issue numbers?

**Comment by pb-programmer** [comment]
Probably issue #186
It's rather hard to replicate and pinpoint the exact problem, but a general overhaul of the "random" function discussed in the thread would be nice. I actually looked into this a little but my fpc skills just aren't up to the task yet.

**Comment by s09bQ5** [comment]
Ok, let me rephrase the question:
Dear person who decides when the next release is ready, what can I do in the next four days to reduce the time until the next release.

**Comment by basisbit** [comment]
@s09bQ5 could you maybe write some short news text for a new USDX version 2020.4 release and create a pull request for https://github.com/UltraStar-Deluxe/ultrastar-deluxe.github.io ? A summary of what changed could then also be used to update the Wikipedia articles about USDX / UltraStar.

**Comment by ePirat** [comment]
I can make macOS builds probably, if needed. Please ping me, if so.

**Comment by s09bQ5** [comment]
@basisbit released v2020.4.0 12 hours ago. He also uploaded a macOS build.

**Comment by ePirat** [comment]
How was the release build done?

**Comment by basisbit** [comment]
1. Get the install script to work again, commit that (did that a while ago)
2. install nsis installer
3. run nsis packaging script
4. upload the result
5. also zip the content of /game/ minus config and *.db files as portable version.
6. upload that portable zip
7. grab the .dmg from travis-ci

**Comment by ePirat** [comment]
Oh note that the dmg from travis CI will only run on macOS Versions equal or higher the version that travis uses.

**Comment by basisbit** [comment]
Any way we could fix that?

**Comment by ePirat** [comment]
Not easily, with the dependencies built by homebrew. (Even building them manually, it's a bit tricky)
So I guess just note that macOS version it will run on on the downloads page so we dont get tons of reports about it not working. It _might_ still work on older versions, but it could crash on startup.

**Comment by s09bQ5** [comment]
We can tell Travis to use an old version of macOS. They have build machines down to macOS 10.10 (released 2014): https://docs.travis-ci.com/user/reference/osx

Homebrew says "10.9–10.12 are supported on a best-effort basis."
